README.md

```diff
@@ -1,94 +1,113 @@
-    * `braingeneers.ml.ephys_dataloader` a high performance pytorch data loader for ephys data.
+## Getting Started
 
-  * `braigeneers.utils`  
-    * `braingeneers.utils.s3wrangler` a wrapper of `awswrangler.s3` for accessing PRP/S3. See section below for the documentation and examples.
-    * `braingeneers.utils.smart_open_braingeneers` a wrapper of `smart_open` for opening files on PRP/S3. See section below for the documentation and examples.
+Welcome to the **Braingeneers Python Utilities** repository! This package collects and provides various Python code and utilities developed as part of the Braingeneers project. The package adheres to the Python Package Authority (PyPA) standards for package structure and organization.
 
-[github]: https://www.github.com/braingeneers/braingeneerspy
+## Contribution
 
-## Installation / upgrade
+We welcome contributions from collaborators and researchers interested in our work. If you have improvements, suggestions, or new findings to share, please submit a pull request. Your contributions help advance our research and analysis efforts.
 
-Most dependencies are optional installations for this package. 
-Below are examples of various installation configurations.
+To get started with your development (or fork), click the "Open with GitHub Codespaces" button below to launch a fully configured development environment with all the necessary tools and extensions.
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/braingeneers/braingeneerspy?quickstart=1)
+
+Instruction on how to contribute to this project can be found in the CONTRIBUTION.md
+
+## Installation
+
+You can install `braingeneerspy` using `pip` with the following commands:
+
+### Install from PyPI (Recommended)
+
+```bash
+pip install braingeneerspy
 ```
-# Typical install (includes `iot`, `analysis`, and `data` access functions, skips `ml`, and lab-specific dependencies): 
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[iot,analysis,data]
 
-# Full install (all optional dependencies included).
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[all]
+### Install from GitHub
 
-# Minimum install (no optional dependencies, good for Raspberry PI builds).
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git
+```bash
+pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git
 ```
 
-### macOS installation note:
-if install fails with ```no matches found: git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[all]```
-wrap quotes around the github address like so 
+### Install from a Wheel (PyPI)
 
+If you prefer to install a pre-built wheel due to resource limitations, you can find the latest release on [PyPI](https://pypi.org/project/braingeneers/).
```

Comment:
Need to specify that users will have to find and download the wheel file.

```
pip download --only-binary :all: --no-deps --dest . --no-cache braingeneers==<version>
```

Reply:
Removed